src/Token/Tokenizer.php

```diff
@@ -587,6 +590,17 @@ private function lexOperator(string $operator): void
         } elseif (':' === $operator && $this->isInTernary()) {
             $bracket = array_pop($this->bracketsAndTernary);
             $this->pushToken(Token::OPERATOR_TYPE, $operator, $bracket);
+        } elseif ('=' === $operator) {
+            $bracket = end($this->bracketsAndTernary);
+            if (false !== $bracket && '(' === $bracket->getValue()) {
```

Comment:
in named arguments for functions and filters (so you may have multiple ones)

Reply:
I just thought about default value for macro.
`{% macro input(name='foo') %}{{ name }}{% endmacro %}`

This one need to be handled differently since it cannot be changed to `:`.

> in named arguments for functions and filters (so you may have multiple ones)

functions and filters (and tests) should work the same (and mutliple named operator too).